GVFS/GVFS.FunctionalTests/FileSystemRunners/PowerShellRunner.cs

```diff
@@ -105,6 +111,11 @@ public override void CreateEmptyFile(string path)
             this.RunProcess(string.Format("-Command \"&{{ New-Item -ItemType file {0}}}\"", path));
         }
 
+        public override void CreateHardLink(string targetPath, string newLinkPath)
+        {
+            Assert.Fail($"{nameof(PowerShellRunner)} does not support {nameof(this.CreateHardLink)}");

```

Comment:
Nit: move this to the base class?

Reply:
Sure, I'll move a default `SupportsHardlinkCreation` to the base as well